### PR TITLE
Govuk asset issue fixes

### PR DESCRIPTION
**Pull request by ljcoulson**
Fixes two issues:

- GOVUK static assets 404s
- Use "GDS Transport" font in Volto admin interface for visual consistency

### CODE REVIEW

**Review by goofballLogic — APPROVED** [review]
Looks good

**Review by nosnickid — CHANGES_REQUESTED** [review]
Had to jump in on this one; I don't think these changes do the trick in prod.

I've got a prod-ish setup locally. I'll make that available in this PR for testing.

When I ran this but forgot to `yarn install` first, I got a webpack copy plugin error. It turns out in main, the copy plugin (@^5.x) already exists. So I assume there's some use of it in razzle.

Looking in the built container, the fonts are still missing

```
root@46c212f40681:/app# find public/
public/
public/robots.txt
public/favicon.ico
```

We need to understand how the copy plugin is used by default and amend `razzle.config.js` when it's running in "build" mode to catch our assets

**Comment by goofballLogic** [comment]
Apologies and thanks for catching @nosnickid

**Comment by nosnickid** [comment]
I was over complicating things with the prod-ish setup. 

Running a `git clean -f public; rm -rf build; yarn build`  from `dd-cms/volto` is enough - running `find build | grep woff` only finds the icons.woff

What I don't understand is why whatever copies icons.woff into assets doesn't catch the fonts we reference too.  

If these fonts are 404'ing in dev too, then we should fix whatever webpack plugin isn't catching our assets
If this is only a prod problem, then tweaking the copy config we added so it copies to `build/public/` will be enough

**Comment by nosnickid** [comment]
Oh, and I think we should remove our copy webpack plugin dep; razzle already has v5 of it and it seems risky upgrading by a major version number

File: volto/razzle.config.js [hunk 1]
```diff
@@ -61,17 +61,15 @@ module.exports.modifyWebpackConfig = ({
...
-          },
-        ],
-      }),
+      new CopyPlugin([
+        {
+          from: path.join(
+            __dirname,
+            '/node_modules/govuk-frontend/govuk/assets',
+          ),
+          to: path.join(__dirname, `/${dev ? '' : 'build/'}public/assets`),
```
Comment (nosnickid) [inline volto/razzle.config.js:70]: Hrm, this is still going to pollute our `public` directory with misc assets every time we run a dev server - git status afterwards shows:

```
On branch govuk-asset-issues
Your branch is up-to-date with 'origin/govuk-asset-issues'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	public/assets/
```

Comment (ljcoulson) [reply volto/razzle.config.js:70]: Can't we just add `public/assets` to gitignore?

Comment (nosnickid) [reply volto/razzle.config.js:70]: Hrm, it's just that the idea of the public dir is for us to manually include any totally static assets, so in principle I don't like ignoring a dir in there.

But it's a bit philosophical; gitignore gets this over the line and looking good for the show and tell, so yeah... let's .gitignore it and move on :+1:

Comment (ljcoulson) [reply volto/razzle.config.js:70]: Fixed

**Comment by nosnickid** [comment]
Eugh sorry I broke your work here; it looks like, despite `yarn list` showing it as a top level dep, copy-webpack-plugin is only installed for razzle.

I'll make a quick PR to include it

**Pull request merged**